src/software/ai/navigator/navigator.cpp

```diff
@@ -2,303 +2,324 @@
+                obstacles.push_back(Obstacle(rectangle));
+                break;
+            case AvoidArea::INFLATED_ENEMY_DEFENSE_AREA:
+                rectangle = world.field().enemyDefenseArea();
+                rectangle.expand(Util::DynamicParameters->getNavigatorConfig()
+                                         ->RobotObstacleInflationFactor()
+                                         ->value() *
+                                     ROBOT_MAX_RADIUS_METERS +
+                                 0.3);  // 0.3 is by definition what inflated means
+                obstacles.push_back(Obstacle(rectangle));
+                break;
+            case AvoidArea::CENTER_CIRCLE:
+                obstacles.push_back(Obstacle::createCircleObstacle(
+                    world.field().centerPoint(), world.field().centerCircleRadius(),
+                    Util::DynamicParameters->getNavigatorConfig()
+                        ->RobotObstacleInflationFactor()
+                        ->value()));
+                break;
+            case AvoidArea::HALF_METER_AROUND_BALL:
+                obstacles.push_back(Obstacle::createCircleObstacle(
+                    world.ball().position(), 0.5,  // 0.5 represents half a metre radius
+                    Util::DynamicParameters->getNavigatorConfig()
+                        ->RobotObstacleInflationFactor()
+                        ->value()));
+                break;
+            case AvoidArea::BALL:
+                obstacles.push_back(
+                    Obstacle::createCircularBallObstacle(world.ball(), 0.06));
+                break;
+            case AvoidArea::ENEMY_HALF:
+                rectangle = Rectangle({0, world.field().totalYLength() / 2},
+                                      world.field().enemyCornerNeg() -
+                                          Point(0, world.field().boundaryYLength()));
+                rectangle.expand(Util::DynamicParameters->getNavigatorConfig()
+                                     ->RobotObstacleInflationFactor()
+                                     ->value() *
+                                 ROBOT_MAX_RADIUS_METERS);
+                obstacles.push_back(Obstacle(rectangle));
+                break;
+            case AvoidArea::FRIENDLY_HALF:
+                rectangle = Rectangle({0, world.field().totalYLength() / 2},
+                                      world.field().friendlyCornerNeg() -
+                                          Point(0, world.field().boundaryYLength()));
+                rectangle.expand(Util::DynamicParameters->getNavigatorConfig()
+                                     ->RobotObstacleInflationFactor()
+                                     ->value() *
+                                 ROBOT_MAX_RADIUS_METERS);
+                obstacles.push_back(Obstacle(rectangle));
+                break;
+            default:
+                LOG(WARNING) << "Could not convert AvoidArea " << (int)avoid_area
```

Comment:
I thought this was blocked on having warnings enforced as errors

Reply:
#779

Reply:
:joy: - sorry, when I said link it, I meant just leave a link in the code with a description of what to change so that the person addressing that ticket can change this as well.